codecs/rotate/rotate.as

```diff
@@ -0,0 +1,63 @@
+export function rotate(inputWidth: i32, inputHeight: i32, rotate: i32): void {
+  const bpp = 4;
+  let offset = inputWidth * inputHeight * bpp;
```

Comment:
`const`?

Reply:
No. So AssemblyScript is not quite like TypeScript in that regard. AssemblyScript uses `const` to denote _compile time_ constants.

Reply:
Ohhhh interesting!

Reply:
this behaviour probably will change in future for consistent with javascript. But currently yes, in this context better use `let` or `var`.